modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/MdfList/MdfList.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import Pagination from './components/Pagination';
+import Table from './components/Table';
+import columns from './components/columns';
+
+const MdfList: any = ({setStep}: {setStep: any}) => {
```

Comment:
```suggestion
const MDFList: any = ({setStep}: {setStep: any}) => {
```

Hi @ytuik, it should be good avoid use any types.

Reply:
Also, it should be good add a interface for use in this component, like:
```
interface IProps {
  setStep: () => void;
}
```